__tests__/identity.js

```diff
@@ -958,6 +958,47 @@ describe('Identity', () => {
             expect(document.getElementsByTagName('body')[0].appendChild).toHaveBeenCalledTimes(1);
             expect(window.openSimplifiedLoginWidget).toHaveBeenCalledTimes(1);
         });
+
+        test('Should emit simplifiedLoginOpened if loaded successfully', async () => {
+            const spy = jest.fn();
+
+            identity._globalSessionService.fetch = jest.fn(() => ({ ok: true, json: () => expectedData }));
```

Comment:
To reduce amount of repetition, what do you think about extracting this mock to 757 line? If you want to mock something different you can always use mockOnce